### ISSUE TITLE
GraphenOS.org

### ISSUE THREAD

**Issue by mkoushan** [ThomazPom/Moz-Ext-UltimaDark#100]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I am on a compatible (Desktop) browser that is up-to-date or explicitly listed as being compatible in the [README](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/blob/master/readme.md)
- [X] I'm reporting that UltimaDark is broken in a novel way that deserves it's own issue meaning it should not be added to [#6](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/issues/6) [#7](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/issues/7) [#8](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/issues/8) or [#94](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/issues/94)
- [X] This site is not listed in an existing issue or in [#6](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/issues/6) [#7](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/issues/7) [#8](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/issues/8) or [#94](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/issues/94)
- [X] I verified that I am on the latest version
- [X] I verified that ImageEdition, Caching, Service Worker Compatibility and the extension itself are all enabled and that I loaded the site cleanly by closing the tab and opening the URL in a new tab or I flushed it completely with [Tab Suspender](https://www.github.com/Hau-Hau/firefox-tab-suspender)
- [X] I tried disabling caching and refreshing the page several times and it is still broken.
- [X] I confirmed that the target site is not on the excluded sites list or a Mozilla domain on Firefox's protected site listin about:config
- [X] I've searched [known issues](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/issues) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [README](https://www.github.com/ThomazPom/Moz-Ext-UltimaDark/blob/master/readme.md) and understand how and why to open an issue.

### Site URL

https://grapheneos.org/

### Provide a description of what about the site is broken and attach screenshots at the bottom

With UltimaDark:
![image](https://github.com/user-attachments/assets/e191a171-1a0d-438b-beda-af0ba66e566d)

Without UltimaDark:
![image](https://github.com/user-attachments/assets/002d636e-5eec-4c54-8fa0-790439f00e6c)

btw I love this extension. KEEP IT UP! :D

### Provide console logs

- [X] Go to about:debugging#/runtime/this-firefox
- [X] Click "Inspect" on UltimaDark
- [X] Clear the log and refresh the page that is broken
- [X] Copy the WHOLE output and insert it below

### Complete Log

```shell
Creating internal property for 
function Element()
 innerHTML background.js:12:21
Creating internal property for 
function ShadowRoot()
 innerHTML background.js:12:21
Creating internal property for 
function CSS2Properties()
 backgroundColor background.js:12:21
Creating internal property for 
function Navigator()
 serviceWorker background.js:12:21
UltimaDark Content script install 
Window https://grapheneos.org/
background.js:2054:15
Creating internal property for 
function Element()
 innerHTML eval:9:21
Creating internal property for 
function ShadowRoot()
 innerHTML eval:9:21
Creating internal property for 
function CSS2Properties()
 backgroundColor eval:9:21
Creating internal property for 
function Navigator()
 serviceWorker eval:9:21
UltimaDark Content script override website 
Window https://grapheneos.org/
eval:11:15
UltimaDark Websites overrides install 
Window https://grapheneos.org/
eval:47:15
Editing function : 
function HTMLObjectElement()
 
function checkValidity()
eval:316:19
Editing function : 
function CSSStyleDeclaration()
 
function setProperty()
eval:316:19
Editing function : 
function CSSStyleSheet()
 
function replace()
eval:316:19
Editing function : 
function CSSStyleSheet()
 
function replaceSync()
eval:316:19
Editing property : 
Array [ Element(), ShadowRoot() ]
 innerHTML eval:255:19
Editing property : 
function Element()
 innerHTML eval:255:19
Editing property : 
function ShadowRoot()
 innerHTML eval:255:19
Editing property : 
Array [ HTMLIFrameElement(), HTMLEmbedElement() ]
 src eval:255:19
Editing property : 
function HTMLIFrameElement()
 src eval:255:19
Editing property : 
function HTMLEmbedElement()
 src eval:255:19
Editing property : 
Array [ HTMLObjectElement() ]
 data eval:255:19
Editing property : 
function HTMLObjectElement()
 data eval:255:19
Editing property : 
Array [ HTMLIFrameElement() ]
 srcdoc eval:255:19
Editing property : 
function HTMLIFrameElement()
 srcdoc eval:255:19
Editing property : 
function Element()
 outerHTML eval:255:19
Editing function : 
function Element()
 
function insertAdjacentHTML()
eval:316:19
Editing function : 
function Element()
 
function setAttribute()
eval:316:19
Editing property : 
function HTMLImageElement()
 src eval:255:19
Editing property : 
function SVGImageElement()
 href eval:255:19
Editing property : 
function HTMLLinkElement()
 href eval:255:19
Wrapping function : 
function SVGSVGElement()
 
function value()
 setAttribute eval:294:19
Wrapping function : 
function HTMLUnknownElement()
 
function value()
 setAttribute eval:294:19
Editing function : 
function Node()
 
function appendChild()
eval:316:19
Editing function : 
function Node()
 
function insertBefore()
eval:316:19
Editing property : 
function Node()
 textContent eval:255:19
Editing property : 
function CSS2Properties()
 background eval:255:19
Editing property : 
function CSS2Properties()
 fill eval:255:19
Editing property : 
function CSSStyleDeclaration()
 cssText eval:255:19
Editing function : 
function CSSStyleSheet()
 
function addRule()
eval:316:19
Editing function : 
function CSSStyleSheet()
 
function insertRule()
eval:316:19
Editing property : 
function CSS2Properties()
 backgroundColor eval:255:19
Editing property : 
function CSS2Properties()
 background-color eval:255:19
Editing property : 
function CSS2Properties()
 color eval:255:19
Editing property : 
Array [ HTMLElement(), SVGElement() ]
 style eval:255:19
Editing property : 
function HTMLElement()
 style eval:255:19
Editing property : 
function SVGElement()
 style eval:255:19
Editing property : 
function HTMLElement()
 innerText eval:255:19
UltimaDark Websites overrides ready 
Window https://grapheneos.org/
 elapsed: 3 eval:312:15
Resolve IDK Vars is ready background.js:2154:15
UltimaDark Content script ready 
Window https://grapheneos.org/
background.js:2162:15
None of the “sha256” hashes in the integrity attribute match the content of the subresource. The computed hash is “j9sG+/1OOrdWNDR2lqe2q0+FtvxWJ17qQeoEQeH+bIw=”. grapheneos.org
```

**Comment by ThomazPom** [comment]
Mistake found and fixed in [1.5.67]. Thank you for your reporting.

**Comment by mkoushan** [comment]
Thanks for the fix. Closing.
